index.html

```diff
@@ -281,7 +294,7 @@ <h3 class="item-title">Skills</h3>
                 </div>
```

Comment:
https://github.com/Bornittah/portifolio/blob/844b2657a209933300854199cdb14513b0914fb8/index.html#L205-L209

- [ ] Please make sure that the images in the `about me` section are displayed properly. This is caused by the incorrect pathing to the assets folder. (You can use a `.` to indicate that the path starts from this file's current path and navigate your way to the assets folder, like so `./assets/images/...<The rest of the path>`, please do this to all of the other images)

- [ ] [OPTIONAL] Please make sure to put background images as a background image for the container, and not as an `img` tag. The background images are images that serve no purpose to the actual page and have no interactions, that's why they shouldn't be presented as an element on the page. You can use the background-image property to set a background for a container in your CSS.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/166683507-3cf6b35d-65d5-4b32-a811-c54ae3e21587.png) | ![image](https://user-images.githubusercontent.com/62290545/166683575-ea473fec-a566-4436-a372-45178ac9aa99.png) |